acquisitions/src/main/resources/thunderjet/consortia/consortia-orders.feature

```diff
@@ -12,15 +12,16 @@ Feature: Open ongoing order
     * def fundId = callonce uuid1
     * def budgetId = callonce uuid2
 
-
   Scenario: init global data
-    #* call login testAdmin
-
     * callonce read('order-utils/inventory.feature')
     * callonce read('order-utils/inventory-university.feature')
     * callonce read('order-utils/configuration.feature')
     * callonce read('order-utils/finances.feature')
     * callonce read('order-utils/organizations.feature')
     * callonce read('order-utils/orders.feature')
 
-    * callonce read('features/open-order-with-locations-from-different-tenants.feature')
+  Scenario: Test open order with locations from different tenants
+    Given call read('features/open-order-with-locations-from-different-tenants.feature')
+
+  Scenario: Test cross-tenant inventory objects creation when working with pieces
+    Given call read("features/pieces-api-test-for-cross-tenant-envs.feature")
```

Comment:
you should have added to the consortia-orders-junit.feature. beacause it will be used to test https://github.com/folio-org/folio-integration-tests/blob/5e1fb7af592063cc214043cf6e3ab0430742cf7b/acquisitions/src/test/java/org/folio/ConsortiaOrdersApiTest.java#L21

Reply:
we can remove this file(consorta-ordrs.feature), it is not being used

Reply:
I think consortia-orders.feature is run from consortia-junit.feature. The first is used to initialize data and run the existing test for mod-orders. For other modules we could have separate consortia-{module}.feature files and encapsulate all the tests related to the module in each of them. Do you suggest it would be better to just use consortia-junit.feature?

Reply:
Well, I didn't know the structure of that, you are right. lets keep in that way.